translations/ko.po

```diff
@@ -213,368 +220,368 @@ msgstr ""
 msgctxt "monster-type-race"
 msgid "Dragon"
-msgstr ""
+msgstr "드래곤족"
 
 #: src\card.ts:79
 msgctxt "monster-type-race"
 msgid "Beast"
-msgstr ""
+msgstr "야수족"
 
 #: src\card.ts:80
 msgctxt "monster-type-race"
 msgid "Beast-Warrior"
-msgstr ""
+msgstr "야수전사족"
 
 #: src\card.ts:81
 msgctxt "monster-type-race"
 msgid "Dinosaur"
-msgstr ""
+msgstr "공룡족"
 
 #: src\card.ts:82
 msgctxt "monster-type-race"
 msgid "Fish"
-msgstr ""
+msgstr "어류족"
 
 #: src\card.ts:83
 msgctxt "monster-type-race"
 msgid "Sea Serpent"
-msgstr ""
+msgstr "해룡족"
 
 #: src\card.ts:84
 msgctxt "monster-type-race"
 msgid "Reptile"
-msgstr ""
+msgstr "파충류족"
 
 #: src\card.ts:85
 msgctxt "monster-type-race"
 msgid "Psychic"
-msgstr ""
+msgstr "사이킥족"
 
 #: src\card.ts:86
 msgctxt "monster-type-race"
 msgid "Divine-Beast"
-msgstr ""
+msgstr "환신야수족"
 
 #: src\card.ts:87
 msgctxt "monster-type-race"
 msgid "Creator God"
-msgstr ""
+msgstr "창조신족"
 
 #: src\card.ts:88
 msgctxt "monster-type-race"
 msgid "Wyrm"
-msgstr ""
+msgstr "환룡족"
 
 #: src\card.ts:89
 msgctxt "monster-type-race"
 msgid "Cyberse"
-msgstr ""
+msgstr "사이버스족"
 
 #: src\card.ts:51
 msgctxt "spell-trap-property"
 msgid "Normal Spell"
-msgstr ""
+msgstr "일반 마법"
 
 #: src\card.ts:52
 msgctxt "spell-trap-property"
 msgid "Continuous Spell"
-msgstr ""
+msgstr "지속 마법"
 
 #: src\card.ts:53
 msgctxt "spell-trap-property"
 msgid "Equip Spell"
-msgstr ""
+msgstr "장착 마법"
 
 #: src\card.ts:54
 msgctxt "spell-trap-property"
 msgid "Quick-Play Spell"
-msgstr ""
+msgstr "속공 마법"
 
 #: src\card.ts:55
 msgctxt "spell-trap-property"
 msgid "Field Spell"
-msgstr ""
+msgstr "필드 마법"
 
 #: src\card.ts:56
 msgctxt "spell-trap-property"
 msgid "Ritual Spell"
-msgstr ""
+msgstr "의식 마법"
 
 #: src\card.ts:57
 msgctxt "spell-trap-property"
 msgid "Normal Trap"
-msgstr ""
+msgstr "일반 함정"
 
 #: src\card.ts:58
 msgctxt "spell-trap-property"
 msgid "Continuous Trap"
-msgstr ""
+msgstr "지속 함정"
 
 #: src\card.ts:59
 msgctxt "spell-trap-property"
 msgid "Counter Trap"
-msgstr ""
+msgstr "카운터 함정"
 
 #: src\card.ts:95
 msgctxt "attribute"
 msgid "EARTH"
-msgstr ""
+msgstr "땅"
 
 #: src\card.ts:96
 msgctxt "attribute"
 msgid "WATER"
-msgstr ""
+msgstr "물"
 
 #: src\card.ts:97
 msgctxt "attribute"
 msgid "FIRE"
-msgstr ""
+msgstr "화염"
 
 #: src\card.ts:98
 msgctxt "attribute"
 msgid "WIND"
-msgstr ""
+msgstr "바람"
 
 #: src\card.ts:99
 msgctxt "attribute"
 msgid "LIGHT"
-msgstr ""
+msgstr "빛"
 
 #: src\card.ts:100
 msgctxt "attribute"
 msgid "DARK"
-msgstr ""
+msgstr "어둠"
 
 #: src\card.ts:101
 msgctxt "attribute"
 msgid "DIVINE"
-msgstr ""
+msgstr "신"
 
 #: src\card.ts:298
 #: src\card.ts:309
 msgctxt "card-embed"
 msgid "Card Text"
-msgstr ""
+msgstr "카드 텍스트"
 
 #: src\card.ts:303
 msgctxt "card-embed"
 msgid "Pendulum Effect"
-msgstr ""
+msgstr "펜듈럼 효과"
 
 #: src\card.ts:324
 msgctxt "card-embed"
 msgid "Card Effect"
-msgstr ""
+msgstr "카드 효과"
 
 #: src\locale.ts:46
 msgctxt "command-option"
 msgid "result-language"
-msgstr ""
+msgstr "출력 언어"
```

Comment:
Although these aren't displayed yet, Discord won't allow spaces for this (see https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-naming)